isis/src/mro/apps/hideal2pds/main.cpp

```diff
@@ -184,6 +184,12 @@ void IsisMain() {
 
   updatePdsLabelTimeParametersGroup(pdsLabel);
   updatePdsLabelImageObject(isisCubeLab, pdsLabel);
+  if (nbits >= 8 && nbits < 16) {
```

Comment:
Why the logic change from the other apps that do this (!=8 or 16)? Does this fix the 8 bit issue you found?

Reply:
I removed the if statement. For nbits==8, it was necessary to explicitly write SAMPLE_BIT_MASK as 2**8-1 in order for the correctly write an 8 bit value to the output label. For nbits=16, it is not necessary to explicitly write SAMPLE_BIT_MASK (it resolves to the correct value without this added bit of code), but I figured it may be appropriate to do so explicitly for the sake of consistency / readability.